src/gui/screen_menu_info.cpp

```diff
@@ -18,5 +18,7 @@ void ScreenMenuInfo::windowEvent(EventLock /*has private ctor*/, window_t *sende
 ScreenMenuInfo::ScreenMenuInfo()
     : ScreenMenuInfo__(_(label)) {
     EnableLongHoldScreenAction();
+#if (PRINTER_TYPE != PRINTER_PRUSA_MINI)
     header.SetIcon(&png::info_16x16);
+#endif //PRINTER_PRUSA_MINI
```

Comment:
This code doesn't seem to be the same thing as before, this effectively disables the line here. Is that on purpose?

Reply:
yes
all other menus in MINI does not use header icon, this one was an exception

I think it is stupid and we should have same icons in MK4/MINI but it is for another discussion